web/modules/custom/portland/portland.module

```diff
@@ -743,8 +742,15 @@ function portland_form_alter(&$form, &$form_state, $form_id)
     // Only set the default field for new Media items
     $form_array_with_group_field = [
         'node_news_form',
+        'node_news_edit_form',
         'node_page_form',
+        'node_page_edit_form',
         'node_event_form',
+        'node_event_edit_form',
+        'node_construction_project_form',
+        'node_construction_project_edit_form',
+        'node_development_permit_form',
+        'node_development_permit_edit_form',
```

Comment:
Can we add this to service and service location as well?

Reply:
Added here